<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickApex3D</class>
 <widget class="QMainWindow" name="PickApex3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mobility Data Display</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>10</number>
       </property>
       <widget class="QWidget" name="display_frame" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>200</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_Input">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Protein selection</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_Protein">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Neutral mass</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="NeutralMass">
             <property name="text">
              <string>22952</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Minimal charge state</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="MinCS">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Mass accuracy (ppm)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="MassAccuracy">
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Maximal charge state</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="MaxCS">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CCS Calibration parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_CCS">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>a</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="Calibration_a">
             <property name="text">
              <string>231.7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>b</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4" colspan="3">
            <widget class="QLineEdit" name="Calibration_b">
             <property name="text">
              <string>118.7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8" colspan="2">
            <widget class="QLineEdit" name="Calibration_X">
             <property name="text">
              <string>0.6262</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="TransferParam">
             <property name="text">
              <string>1.41</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" colspan="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Pusher rate (Âµs)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="PusherDelay">
             <property name="text">
              <string>110</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6" colspan="3">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Gas mass</string>
             </property>
            </widget>
           </item>
           <item row="1" column="9">
            <widget class="QLineEdit" name="Gas_mass">
             <property name="text">
              <string>28</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="Output_Data">
          <property name="rowCount">
           <number>10</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Current series</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="selectSeries">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="Output_Series">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rowCount">
           <number>7</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoad_Data_File"/>
    <addaction name="actionSave_processed"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionNext_series"/>
    <addaction name="actionPrevious_series"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Data_File"/>
   <addaction name="actionSave_processed"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionLoad_Data_File">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>../apex3DNavigator</normaloff>../apex3DNavigator</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Data File</string>
   </property>
   <property name="toolTip">
    <string>Load </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_processed">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>../apex3DNavigator</normaloff>../apex3DNavigator</iconset>
   </property>
   <property name="text">
    <string>&amp;Save processed</string>
   </property>
   <property name="toolTip">
    <string>Save the processed data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../apex3DNavigator</normaloff>../apex3DNavigator</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit program without saving</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNext_series">
   <property name="text">
    <string>&amp;Next series</string>
   </property>
   <property name="toolTip">
    <string>Choose next series</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionPrevious_series">
   <property name="text">
    <string>&amp;Previous series</string>
   </property>
   <property name="toolTip">
    <string>Select previous series</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
